Source/WebCore/dom/ToggleEvent.h

```diff
@@ -56,6 +68,20 @@ class ToggleEvent final : public Event {
     String m_newState;
 };
 
+class ToggleEventTask final: public RefCounted<ToggleEventTask> {
```

Comment:
Seems to me this would be better in a separate header file. Almost no code needs to know about both `ToggleEventTask` and `ToggleEvent`.